src/universal/components/ListFilterSet/ListFilterSet.js

```diff
@@ -3,6 +3,8 @@ import cssModules from 'react-css-modules';
 import styles from './ListFilterSet.scss';
 
 @cssModules(styles)
+// for the decorators
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
Yeah, it only pertains to decorators because of a combination of rules.

The component fails linting because the ESLint wants it to be a functional component (as it should given the component's structure), but we can't make it a functional component because functions can't be decorated. So the only work around is to disable the rule.